### PR TITLE
slof_device_tree: Refactoring code 

### PR DESCRIPTION
**Pull request by wangxianxian**
Get the last line of output of get_info();
Create a new function compare_dev_tree() to reduce code redundancy;
Decode output of process.system_output() for host_system_id;

ID: 1704652

Signed-off-by: Xianwang <xianwang@redhat.com>

### CODE REVIEW

**Comment by wangxianxian** [comment]
Test PASS.

**Comment by wangxianxian** [comment]
@YongxueHong 
please review it, thanks

File: qemu/tests/slof_device_tree.py [hunk 1]
```diff
@@ -57,29 +56,14 @@ def check_nonexist_aliases(vm_session, devices):
...
-        if guest_system_id != vm.get_uuid():
-            raise exceptions.TestFail("Guest system id does not match to uuid")
-
-        guest_uuid = get_info(session, "vm,uuid")
-        if guest_uuid != vm.get_uuid():
-            raise exceptions.TestFail(
-                "Guest uuid does not match to expected id.")
-
+        compare_dev_tree("system-id", vm.get_uuid())
+        compare_dev_tree("vm,uuid", vm.get_uuid())
```
Comment (YongxueHong) [inline qemu/tests/slof_device_tree.py]: Better :
```
        uuid = vm.get_uuid()
        compare_dev_tree("system-id", uuid)
        compare_dev_tree("vm,uuid", uuid)
```

File: qemu/tests/slof_device_tree.py [hunk 2]
```diff
@@ -47,8 +47,7 @@ def check_nonexist_aliases(vm_session, devices):
         error_context.context(
             "Checking whether aliases file is indeed nonexisting", logging.info)
         if status == 0:
-            raise exceptions.TestFail(
-                "Nonexist cdrom aliases check failed.")
+            test.fail("Nonexist cdrom aliases check failed.")
```
Comment (YongxueHong) [inline qemu/tests/slof_device_tree.py]: if there is no return value, pls update the function's docstring.

File: qemu/tests/slof_device_tree.py (hunk 1 above)
Comment (wangxianxian) [reply qemu/tests/slof_device_tree.py]: I have updated it.

File: qemu/tests/slof_device_tree.py (hunk 2 above)
Comment (wangxianxian) [reply qemu/tests/slof_device_tree.py]: There should be two return value for status: zero or none zero, if it is none zero, the result is expected.

**Comment by hereischen** [comment]
Acked-by: Haotong Chen hachen@redhat.com

**Comment by YongxueHong** [comment]
Thanks all.

**Pull request merged**